src/main/mondrian/spi/impl/JdbcDialectImpl.java

```diff
@@ -331,26 +331,10 @@ public void quoteIdentifier(final String val, final StringBuilder buf) {
             return;
         }
 
-        int k = val.indexOf('.');
```

Comment:
@mbatchelor This logic scanning for the '.' is pretty ancient, and I think it pre-dates the "schema" attribute, which would be the preferred way to specify the "owner".  Given that periods are allowed characters in table and column names in many databases, we can't safely extract the schema part based on the presence of the dot.

Including @lucboudreau in case he has any concerns.